packages/fast-development-site-react/app/components/index.ts

```diff
@@ -0,0 +1,11 @@
+import Button from "./button";
+export { Button };
+export * from "./button";
+
+import Paragraph from "./paragraph";
+export { Paragraph };
+export * from "./paragraph";
+
+import Span from "./span";
+export { Span };
+export * from "./span";
```

Comment:
Is this a necessary addition for this PR? This seems like more additional overhead to maintain for components that only exist to test on the dev site.

Reply:
See above comment.